ripe/atlas/tools/commands/stream.py

```diff
@@ -50,12 +63,22 @@ def add_arguments(self):
                  "appropriate renderer will be selected."
         )
 
+    def _get_request_auth(self):
+        if self.arguments.auth:
+            return conf["authorisation"]["fetch_aliases"][self.arguments.auth]
+        else:
+            return conf["authorisation"]["fetch"]
+
     def run(self):
 
         try:
             measurement = Measurement(
-                id=self.arguments.measurement_id, user_agent=self.user_agent)
-        except APIResponseError:
+                id=self.arguments.measurement_id, user_agent=self.user_agent,
+                key=self._get_request_auth)
+        except APIResponseError as e:
+            if "error" in e.args[0]:
+                if "detail" in e.args[0]["error"]:
+                    raise RipeAtlasToolsException(e.args[0]["error"]["detail"])
             raise RipeAtlasToolsException("That measurement does not exist")
```

Comment:
I preferred a more user friendly message here instead of an error dump like `{u'error': {u'status': 403, u'code': 104, u'detail': u'Authentication credentials were not provided.', u'title': u'Forbidden'}}`, but no problem to use the whole `e.args[0]`.